src/data/uts-isf/uts-isf_oecm_2024.json

```
@@ -0,0 +1,101 @@
+{
+  "$schema": "http://pathways.rmi.org/schema/pathwayMetadata.v1.json",
+  "id": "uts-isf_oecm_2024",
+  "publication": {
+    "title": {
+      "full": "One Earth Climate Model 2024",
+      "short": "OECM"
+    },
+    "subtitle": "Science-based Target Setting for the Finance Industry",
+    "author": ["Teske, S.", "Feenstra, S.", "Rispler, J.", "Niklas, S."],
+    "publisher": {
+      "full": "Institute for Sustainable Futures, University of Technology Sydney",
+      "short": "UTS ISF"
+    },
+    "year": 2024,
+    "month": 9,
+    "city": "Sydney",
+    "license": "CC0 1.0",
+    "links": [
+      {
+        "description": "Report",
+        "url": "https://www.uts.edu.au/globalassets/sites/default/files/2024-10/One-Earth-Climate-Model-2024-Final.pdf"
+      },
+      {
+        "description": "Dataset",
+        "url": "https://datadryad.org/dataset/doi:10.5061/dryad.cz8w9gj82"
+      }
+    ]
+  },
+  "name": { "full": "1.5°C Pathway", "short": "1.5°C" },
+  "description": "Global 1.5°C pathway driven by massive rapid deployment of existing low-carbon technology.",
+  "geography": [
+    "Global",
+    "AR",
+    "AU",
+    "BR",
+    "CA",
+    "CN",
+    "FR",
+    "DE",
+    "IN",
+    "ID",
+    "IT",
+    "JP",
+    "MX",
+    "RU",
+    "SA",
+    "ZA",
+    "KR",
+    "TR",
+    "GB",
+    "US"
+  ],
+  "pathwayType": "Normative",
+  "carbonBudget": 400,
+  "modelTempIncrease": 1.5,
+  "modelYearStart": 2019,
+  "modelYearEnd": 2100,
+  "modelYearNetzero": 2050,
+  "ssp": "SSP1",
+  "sectors": [
+    {
+      "name": "Power",
+      "technologies": [
+        "Coal",
+        "Oil",
+        "Gas",
+        "Nuclear",
+        "Hydro",
+        "Wind",
+        "Solar",
+        "Biomass",
+        "Other"
+      ]
+    }
```

```suggestion
    { "name": "Buildings", "technologies": [] },
    { "name": "Industry", "technologies": [] },
    { "name": "Steel", "technologies": [] },
    { "name": "Cement", "technologies": [] },
    { "name": "Chemicals", "technologies": [] },
    { "name": "Coal Mining", "technologies": [] },
    { "name": "Oil (Upstream)", "technologies": [] },
    { "name": "Gas (Upstream)", "technologies": [] },
    {
      "name": "Power",
      "technologies": [
        "Coal",
        "Oil",
        "Gas",
        "Nuclear",
        "Hydro",
        "Wind",
        "Solar",
        "Biomass",
        "Other"
      ]
    },
    { "name": "Transport", "technologies": [] },
    { "name": "Road transport", "technologies": [] },
    { "name": "Automotive", "technologies": [] },
    { "name": "Aviation", "technologies": [] },
    { "name": "Rail", "technologies": [] },
    { "name": "Shipping", "technologies": [] },
    { "name": "Other", "technologies": [] }
```

An update from the draft tab did not make it to the final tab. The pathways covers more sectors.